docs/src/math_framework.md

```diff
@@ -74,6 +74,58 @@ while the _contravariant basis_ is the opposite: gradient in ``x`` of the coordi
 \mathbf{e}^i = \nabla_x \xi^i
 ```
 
+Here is an intuitive example of how the _contravariant basis_ and _covariant basis_ changes when a coordinate is being transformed.
+
+![Different bases supported in ClimaCore.jl](controva_cova.png)
+
+Start with the _contravariant basis_, which is the case that most of us is more familier. As it can be seen in the figure below, we have 2D vector $\vec{a}$, 
+and two unit vectors $\vec{e_{1}}$ and $\vec{e_{2}}$, served as two basis. Thus we can represent vector $\vec{a}$ as:
```

Comment:
Even this sentence can be rephrased a bit (as it has some grammar slips/typos). Also, the figure is above of this text, not below.